src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -72,7 +76,6 @@ const hostDiscoveryStepValidationsMap: WizardStepValidationMap = {
   cluster: {
     groups: [],
     validationIds: [
-      'sufficient-masters-count',
```

Comment:
We need to understand why this condition has been removed, IMO if this validation should still be required.

Reply:
I added back, otherwise it would allow us to continue to Networking with an invalid number of hosts.